ee/clickhouse/views/test/__snapshots__/test_clickhouse_trends.ambr

```diff
@@ -505,7 +505,7 @@
   SELECT groupArray(value)
   FROM
     (SELECT replaceRegexpAll(JSONExtractRaw(properties, 'key'), '^"|"$', '') AS value,
-            count(*) as count
+            count(DISTINCT pdi.person_id) as count
```

Comment:
Any reason we're doing this vs `uniqExact`? Seems like this will be slower.

Reply:
This already is `uniqExact` (see https://clickhouse.com/docs/en/operations/settings/settings/#settings-count_distinct_implementation)

Reply:
Do you consider it that bad? 😄 They're the same
<img width="469" alt="Screenshot 2022-12-05 at 15 12 15" src="https://user-images.githubusercontent.com/4550621/205657808-51edd097-39ac-450e-8632-b97f8c8fbbfe.png">